components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -137,7 +137,9 @@ internal class DisplayToolbar(
                 anchor = this,
                 orientation = BrowserMenu.determineMenuOrientation(toolbar))
 
-            emitOpenMenuFact()
+            val isCustom = (menuBuilder?.extras?.get("customTab")) as? Boolean
+
+            emitOpenMenuFact(isCustom == true)
```

Comment:
No need to do anything special for "customTab". Let's just pass along the whole map and then add it as metadata in `emitOpenMenuFact`.